src/styles/components/_definition.scss

```diff
@@ -4,50 +4,99 @@ $breakpoint-background-image-size: 75rem;
   color: var(--foreground-color);
   width: 100%;
   position: relative;
-  margin: 2.5rem 0 0; // top: 40px; left&right: 0; bottom: 0;
+  padding: 8rem 0;
 
-  @media (min-width: $breakpoint-scale-up) {
-    margin-bottom: -8.5625rem; // compensates for code example padding
+  &::after {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: calc(-50vw + 49.5%); // Forbidden Zastrow Knowledge
```

Comment:
Is it terrible to push this entirely to 50%? I'm getting a tiny li'l black border between the pink text, and the actual edge of the window:  

<img width="82" alt="Screen Shot 2022-12-15 at 10 48 03 AM" src="https://user-images.githubusercontent.com/69602589/207909917-1550d5ac-83f3-4f91-b5ee-2b96cb198acf.png">

Reply:
Oh that's weird. 🤔

Reply:
Dustin pointed out that the changes are probably due to the inclusion of the scrollbar width in 100vw. I've added 0.6rem to the `right` value to compensate, though I am a bit worried it's different on Windows devices.

Reply:
Ok. I'll pull the deploy preview into browserstack and see if I can find anything...